dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -45,22 +57,18 @@ cmd_config() {
         log "rm -rf \"${CHE_HOST_INSTANCE}/dev\" >> \"${LOGS}\""
         rm -rf "${CHE_CONTAINER_INSTANCE}/dev"
     fi
+
+    if [[ ! -d $(echo ${CHE_CONTAINER_DEVELOPMENT_REPO}/${CHE_ASSEMBLY_IN_REPO}) ]]; then
```

Comment:
this could not be in docker.sh because it is used by both che and codenvy where assemblies are different we need custom logic for both and if we will move it `docker.sh` it will be same for both which does not work.